src/Uno.UWP/Gaming/Input/GamepadReading.cs

```diff
@@ -0,0 +1,49 @@
+namespace Windows.Gaming.Input
+{
+	/// <summary>
+	/// Represents the current state of the gamepad.
+	/// </summary>
+	public partial struct GamepadReading
+	{
+		/// <summary>
+		/// Time when the state was retrieved from the gamepad.
+		/// </summary>
+		public ulong Timestamp;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5499310)